<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPreferencesDialog</class>
 <widget class="QDialog" name="GlobalPreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of the window that contains the preferences and settings that affect the entire app (global settings)">Global Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="preferencesSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="preferenceCategoryList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
       <property name="toolTip">
        <string extracomment="Title of the general or miscellaneous options in the app settings"/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/wrench.png</normaloff>:/icons/wrench.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Display</string>
       </property>
       <property name="toolTip">
        <string extracomment="Title of the display-related settings section - the window size, text size, screen size options in the app settings"/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/monitor_edit.png</normaloff>:/icons/monitor_edit.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Performance</string>
       </property>
       <property name="toolTip">
        <string extracomment="Title of the performance related options in the app settings"/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/processor.png</normaloff>:/icons/processor.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Language</string>
       </property>
       <property name="toolTip">
        <string extracomment="Title of the language and locale related options in the app settings"/>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/locate.png</normaloff>:/icons/locate.png</iconset>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="preferencePageStack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="generalPreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="generalTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string extracomment="Title text for a general app preferences or settings section">General</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="generalTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QGroupBox" name="startupOptionsGroup">
              <property name="title">
               <string extracomment="Title for the on startup section of app settings. Settings in this section are applied when the app launches or first starts up">Start Up Settings</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_27">
               <item>
                <widget class="QCheckBox" name="geometrizeLogo">
                 <property name="text">
                  <string extracomment="Label text for a checkbox option that controls whether the app icon is animated (geometrized) when the app is launched">Animate app logo on launch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="populateRecents">
                 <property name="text">
                  <string extracomment="Label text for a checkbox option that controls whether the list of recent files or items is filled up when the app is launched">Load recent items list</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="populateTemplates">
                 <property name="text">
                  <string extracomment="Label text for a checkbox option that controls whether a list of sample images is loaded up when the app is launched">Load sample images</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="displayPreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="displayTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string extracomment="Title text for a display-related (screen-size, font size etc) app preferences or settings section">Display</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="displayTitleSpacer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QGroupBox" name="launchPageDisplayOptionsGroup">
              <property name="title">
               <string extracomment="Title for the app settings section that contains options that affect the launch or launcher page of the app">Launch Page Settings</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <widget class="QCheckBox" name="showLaunchWindowConsoleByDefault">
                 <property name="text">
                  <string extracomment="Label text next to a checkbox that controls whether a script console window will be shown or hidden">Show script console</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="imageTaskDisplayOptionsGroup">
              <property name="title">
               <string extracomment="Title for the app settings section that contains options that affect the area of the app that turns images into shapes (the image geometrizer)">Image Geometrizer Settings</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QCheckBox" name="showVectorResultsByDefault">
                 <property name="text">
                  <string extracomment="Label text next to a checkbox that controls whether a vector-based (SVG-based) image is shown or hidden">Show vector image view</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showPixmapResultsByDefault">
                 <property name="text">
                  <string extracomment="Label text next to a checkbox that controls whether a raster-based (pixmap-based) image is shown or hidden">Show raster image view</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showImageTaskConsoleByDefault">
                 <property name="text">
                  <string extracomment="Label text next to a checkbox that controls whether a script console window will be shown or hidden">Show script console</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showScriptEditorsByDefault">
                 <property name="text">
                  <string extracomment="Label text next to a checkbox that controls whether a window containing controls for editing and manipulating script files is shown or hidden">Show script editor window</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="performancePreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="performanceTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string extracomment="Title text for performance-related (processing speed) app preferences or settings section">Performance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="performanceTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QGroupBox" name="imageResizingGroup">
              <property name="title">
               <string extracomment="Title text for a group of settings that control how images are scaled down or made smaller (downscaled)">Image Downscaling</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QCheckBox" name="imageResize">
                   <property name="text">
                    <string extracomment="Label text next to a checkbox that controls whether an app feature is enabled or disabled">Enabled</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="resizeThresholdLabel">
                   <property name="toolTip">
                    <string/>
                   </property>
                   <property name="text">
                    <string extracomment="A text label next to a pair of values that describe the maximum size an image can be without the image having to be scaled down and made smaller">Resize Threshold</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_19">
                   <item>
                    <layout class="QFormLayout" name="formLayout_2">
                     <property name="formAlignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="resizeWidthLabel">
                       <property name="text">
                        <string extracomment="Label text next to a value that indicates the maximum allowed width of an image in pixels">Max Width</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QSpinBox" name="resizeWidth">
                       <property name="maximum">
                        <number>65536</number>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="resizeHeightLabel">
                       <property name="text">
                        <string extracomment="Label text next to a value that indicates the maximum allowed height of an image in pixels">Max Height</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QSpinBox" name="resizeHeight">
                       <property name="maximum">
                        <number>65536</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="threadingOptionsGroup">
              <property name="title">
               <string extracomment="Title text for a group of multithreading-related settings. This is threading in the sense of CPU threads (not clothes!)">Multithreading</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="threadsPerImageTaskLabel">
                     <property name="text">
                      <string extracomment="A text label next to a value that describes the maximum number of CPU threads the computer can use when recreating an image using shapes">Max Threads Per Image</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="maxThreadsPerImageTask">
                     <property name="maximum">
                      <number>128</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="languagePreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="languageTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string extracomment="Title text for a language and locale app settings section">Language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="languageTitleSpacer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="geometrize::dialog::LanguageSelectWidget" name="widget" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="closeButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>geometrize::dialog::LanguageSelectWidget</class>
   <extends>QWidget</extends>
   <header>dialog/languageselectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
